app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date and expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine is outdated.")
+
+        if "wearing_a_mask" in visitor and not visitor["wearing_a_mask"]:
```

Comment:
If visitor doesn't contain `wearing_a_mask `, can he visit to a cafe? I think, you must modify your code.
```suggestion
        if not visitor.get("wearing_a_mask", False):
```